Editor/EditorCore/Experimental.cs

```diff
@@ -26,6 +26,11 @@ internal static bool experimentalFeaturesEnabled
         [UserSettingBlock("Experimental")]
         static void ExperimentalFeaturesSettings(string searchContext)
         {
+#if PROBUILDER_EXPERIMENTAL_FEATURES
+            //After reset ScriptingDefineSymbol is still active and needs to be deactivated
+            if(!s_experimentalFeatureEnabled.value)
+                ScriptingSymbolManager.RemoveScriptingDefine(k_ExperimentalFeaturesEnabled);
```

Comment:
Is there a callback in Settings Manager for when a pref, or all prefs, are reset?

Reply:
No I cheked last time and didn't see any, I might have missed that however...

Reply:
@karl- I just checked on the settings manager repo and it doesn't seems to have callbacks neither for property changed nor reset